ModificationResponse/Editor/ModificationResponseExampleWindow.cs

```diff
@@ -0,0 +1,94 @@
+﻿using UnityEditor;
+using UnityEngine;
+
+namespace Unity.Labs.SuperScience
+{
+    public class ModificationResponseExampleWindow : EditorWindow
+    {
+        const double k_TimeToResponse = 0.3;
+        const float k_ColorRectPadding = 4f;
+        const string k_HelpMessage = "This window shows the average color from all ColorContributors in the Scene. " +
+            "It is updated on a delayed response to property modifications to ColorContributors. " +
+            "We use averaging colors here for the sake of simplicity - in practice this delayed response pattern " +
+            "is most useful for operations with a lot of overhead.";
+
+        double m_TimeOfLastChange = -1;
+        Color m_AverageColor;
+
+        [MenuItem("Window/Modification Response Example")]
```

Comment:
Please merge this branch with the latest `master` state, and put these windows under a SuperScience group. #8 updated the other windows to do the same, and it helps reduce clutter in the Window menu.